force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -101,19 +158,36 @@ public with sharing class ClassesObjectsMethods {
      * @return The number of vowels in the string.
      */
     public static Integer countVowels(String str) {
-        // Initialize the result Integer
-
-        /*
-        for () {
+        //Initialize the result Integer
+        Integer count = 0;
+        List<String> vowels = new List<String>{'a','e','i','o','u'};
+        // // Loop through the string
+        // // Compare the character to the vowels set
+        // // If the character is a vowel, increment the result integer
+        // // return the result Integer
+       
+        for (Integer i = 0; i < str.length(); i++)
+        {  
+            String eachLetter = str.substring(i, i + 1);
+            if (vowels.contains(eachLetter))
+            {
+                count++;
+            }
+                
+        }       
             // get individual characters from the string - hint use substring or split
             // check if the character is a vowel
             // if it is a vowel, increment the result Integer
-        }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        //return count; // Replace null with the variable you used to store the result
+        return count;
     }
 
+    //String ch = str.substring(i, i + 1);
+        
+    // Check if the character is a vowel
+    //if (ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u') {
+        
```

Comment:
Remove commented-out code prior to submission. This will keep your code clean and shorter (and avoid clutter for your reviewer).